### ISSUE TITLE
[BUG] colab notebook cuGraph example is broken

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cugraph#398]
@BradReesWork using the latest cuGraph nightly conda packages, the example snippet gives an error.

Did the API change?

```
import cugraph, cudf
import gzip, io, requests
from collections import OrderedDict

# download some data
url="https://snap.stanford.edu/data/soc-Epinions1.txt.gz"
content = gzip.decompress(requests.get(url).content).decode()

cols = ["src", "dst"]
dtypes = OrderedDict([ ("src", "int32"), ("dst", "int32")])
# read the CSV data from memory buffer
gdf = cudf.read_csv(io.StringIO(content), names=cols, delimiter='\t', dtype=list(dtypes.values()), skiprows=4)

# create a Graph 
G = cugraph.Graph()
G.add_edge_list(gdf["src"], gdf["dst"])

# Call Pagerank on the graph to get weights to use:
pr_df = cugraph.pagerank(G)

# find the max page rank value - there could be more than one with the max score
pr_max = pr_df['pagerank'].max()

pr_filtered = pr_df.query('pagerank >= @pr_max')
    
for i in range(len(pr_filtered)):
    print("PageRank: top vertex is " + str(pr_filtered['vertex'][i]) + 
        " with score of " + str(pr_filtered['pagerank'][i]))  

# Call weighted Jaccard using the Pagerank scores as weights:
df = cugraph.jaccard_w(G, pr_df['pagerank'])

max_coeff = df['jaccard_coeff'].max()
j_gdf = df.query('jaccard_coeff >= @max_coeff')

for i in range(len(j_gdf)):
    print("Weighted Jaccard Similarity: Vertices " + str(j_gdf['source'][i]) + 
      " and " + str(j_gdf['destination'][i] ) + 
      " are most similar with score: " + str(df['jaccard_coeff'][i]))
```
Result:
```
PageRank: top vertex is 18 with score of 0.004534927
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
AttributeError: 'NoneType' object has no attribute '_column'
Weighted Jaccard Similarity: Vertices 12651 and 12647 are most similar with score: 0.15591398
Exception ignored in: 'cugraph.utilities.column_utils.get_gdf_column_view'
AttributeError: 'NoneType' object has no attribute '_column'
```

**Comment by randerzander** [comment]
It appears that `cugraph.jacard_w` internally tries to access an attribute of gdf_column_view that no longer exists.

**Comment by Iroy30** [comment]
https://github.com/rapidsai/cugraph/blob/9241f9921fb39ccebb9ff3325443eaa4fee5ef29/python/cugraph/link_prediction/wjaccard.py#L79 needs to be changed to 
df = wjaccard_wrapper.jaccard_w(input_graph.graph_ptr, weights, first, second)

**Comment by afender** [comment]
#414 should fix this

**Comment by randerzander** [comment]
Closed by #414
